fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaGenerator.java

```diff
@@ -717,6 +717,9 @@ public Table addLogicalResourceTags(PhysicalDataModel pdm) {
                 })
                 .build(pdm);
 
+        tbl.addTag(SCHEMA_GROUP_TAG, FHIRDATA_GROUP);
+        this.procedureDependencies.add(tbl);
+
         pdm.addTable(tbl);
         pdm.addObject(tbl);
```

Comment:
Do you think it's worth adding a check to make sure there's at least one tag on the object? A better change (although a lot more work) would be to make tagging part of the builder and then we could make it mandatory.

Reply:
I've added the checks in place to tests, and modified where the tags are missing. 
The build will fail without it.